website/src/writing-php-code/narrowing-types.md

```diff
@@ -200,6 +200,28 @@ public function getName(): ?string
 }
 ```
 
+By default `@phpstan-assert-if-true` also makes assumptions about `false` return value, in case this is not desired use `=` operator before the type.
```

Comment:
Please explain this in more depth. Besides assumptions about the opposite situation, `=` also helps you to get rid of some error messages, like `Call to function x() will always evaluate to true.`.

Reply:
I did add additional details, however please check if what I wrote is actually true, my understanding of Phpstan is not deep. This feels a bit too verbose or maybe long, but at the same time it does explain the feature clearly with an example, so maybe it is for the best.